definitions/crds/kyverno.io_policies.yaml

```diff
@@ -323,11 +323,12 @@ spec:
                               type: string
                           type: object
                         data:
+                          allOf:
```

Comment:
Is this the only change to the CRD? Adding this new tag `allOf` means all previous generate policies with `data` defined are no longer valid, how should we handle backward compatibility?

Reply:
I'll test this out over the weekend and get back to you :+1: